pom.xml

```diff
@@ -1300,7 +1300,12 @@
             <dependency>
                 <groupId>commons-pool</groupId>
                 <artifactId>commons-pool</artifactId>
-                <version>1.6</version>
```

Comment:
there is a cq already for 1.5.4, and 1.6 - we could open one for 1.5.3. Are we actually still using this, or could we mark it provided scope?